plugins/MusicPlayer/player.js

```diff
@@ -312,7 +312,7 @@ exports.dequeue = {
     * Pause command.
     *
     * @param msg Original message.
-    * @param suffix Command suffix.
+    * @param suffix Command s>uffix
```

Comment:
Changes to words from english to another language doesn't warrant a pull request to the main repo as majority of the users are english speakers. If you want you can make a translation file in yml or json format and can send it over for use in making this bot multilingual.